tests/e2e/scenarios/ha/README.md

```diff
@@ -0,0 +1,105 @@
+# Skupper HA (High Availability) Test
+
+This README provides an overview of the "Skupper HA Test". 
+
+## Overview
+
+This test creates two Skupper sites with the HA feature enabled, this creates an additional instance of the skupper router in each skupper site.
+A backend application is deployed in the east_ha namespace and exposed via Skupper in the west_ha namespace.
```

Comment:
not exactly in the `east_ha` and `west_ha`, but using namespaces with the `east-ha` and `west-ha` suffix (and with dashes, instead of underscores, is that right?).